test/e2e.sh

```diff
@@ -27,7 +27,7 @@ else
 fi
 
 if [[ "$KUBECONFIG" != "$HOME/.kube/kind-config-kind" ]] ; then
-  $BIN ca generate --name ingress-ca --cert-path .certs/ingress-ca-crt.pem --private-key-path .certs/ingress-ca-key.pem --password foobar  --expiry 1
+  $BIN ca generate --name ingress-ca --cert-path .certs/root-ca.crt --private-key-path .certs/root-ca.key --password foobar  --expiry 1
```

Comment:
Why the change in filename ?

Reply:
Happy to revert.
This got back-ported to `common.yaml` from the platform config I generated to get a single master cluster for audit test.

about extensions: `.key` and `.crt` seems more "standard" (if at least more common) for CAs.

about names - I was focusing on the root CA.

Ideally I should change the name from `ingress-ca` as well.

What's the verdict?
Back to `ingress-ca-crt.pem` and `ingress-ca-key.pem`,
or 
change the name to `root-ca` as well?

Reply:
+1 for the rename to .key and .crt

I think let's leave ingress-ca and add root-ca - that way we can pick any issues with using the wrong ca.

Reply:
Cool done in e9bfe90e0608705a5f75e34c5b24624674832d61